frontend/src/core/wasm/bridge.ts

```diff
@@ -187,6 +187,11 @@ export class PyodideBridge implements RunRequests, EditRequests {
     return null;
   };
 
+  sendCopy: EditRequests["sendCopy"] = async () => {
+    // TODO: Implement copy with pyodide!
+    return null;
```

Comment:
We can throw not implemented. And I think we can hide this action in the notebook dropdown when isWasm is true